tests/introspection/tests.py

```diff
@@ -171,8 +171,8 @@ def test_get_indexes(self):
         with connection.cursor() as cursor:
             indexes = connection.introspection.get_indexes(cursor, Article._meta.db_table)
         if connection.features.can_introspect_index_type:
-            self.assertIn('type', indexes['reporter_id'])
-            indexes['reporter_id'].pop('type')
+            index_type = indexes['reporter_id'].pop('type', None)
+            self.assertEqual(index_type, 'btree')
```

Comment:
Interesting point. Actually, we need some way to map a canonical list of index names to `Index` classes. For example, 'btree' maps to `Index` for PostgreSQL. Should we use `Index.suffix` for that?